src/SUMMARY.md

```diff
@@ -38,12 +38,12 @@
       - [Goals and clauses](./traits/goals-and-clauses.md)
       - [Equality and associated types](./traits/associated-types.md)
       - [Implied bounds](./traits/implied-bounds.md)
+      - [Well-formedness checking](./traits/wf.md)
```

Comment:
Could you put this chapter just after "lowering rules"? It's intended to be a "complete reference" chapter and I think it's better for this one to read it after being comfortable with the full set of lowering rules.

Reply:
done.